Mathlib/RingTheory/MvPolynomial/Symmetric.lean

```diff
@@ -23,15 +26,24 @@ We also prove some basic facts about them.
 
 * `MvPolynomial.esymm`
 
+* `MvPolynomial.hsymm`
```

Comment:
Can you add a (very) short description of each declaration?

Reply:
Aren't they already there?

Reply:
I mean like in line 23.

Reply:
Ah I see! Sorry!